hosts/ficolo-common.nix

```diff
@@ -1,22 +1,9 @@
 # SPDX-FileCopyrightText: 2022-2024 TII (SSRC) and the Ghaf contributors
 # SPDX-License-Identifier: Apache-2.0
 #
-{
-  self,
-  lib,
-  machines,
-  ...
-}:
-{
-  imports = [ self.nixosModules.service-monitoring ];
-
+_: {
```

Comment:
`_: ` in the beginning of line 4 is unnecessary leftover and should be removed

Reply:
Fixed